src/index.html

```diff
@@ -9,6 +9,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a href="./index.html" class="header-logo">
+        <img src="./images/logo.png" alt="Moyo logo">
+      </a>
+      <nav class="nav-block">
+        <ul class="nav-list">
+          <li class="nav-item">
```

Comment:
Please, check if the code follows all code style rules 
![image](https://github.com/mate-academy/layout_moyo-header/assets/46566190/8fc2d56c-9e16-4c4b-a515-25e8f56e883a)